src/migrations/migration.js

```diff
@@ -0,0 +1,40 @@
+const createUser = `
+    DROP TABLE IF EXISTS users;
```

Comment:
You can drop table/database for your test DB, not your development or staging DB. Dropping table will remove all user data in it and that is not a good thing for when you have users.

A good way is to run the query if the table do not exist already.